examples/module1/lesson3/shop-rtk-start/components/Product.tsx

```diff
@@ -11,8 +11,17 @@ const Product = ({ product }: { product: ProductType }) => {
   const dispatch = useAppDispatch();
   const { id, image, category, title, price } = product;
 
+  const addProductToCart = () => {
+    dispatch(addToCart(product));
+    toast.success(`${title} added to cart!`);
```

Comment:
Nie jestem pewien czy to jest dobre miejsce do wrzucenia tego toast'a 🤔. Jeśli trzeba byłoby wysłać call do API, to nie wiemy czy taka operacja się powiedzie, także wyświetlenie toasta, mogłoby wprowadzić użytkownika w błąd :) podejrzewam, że rtk wprowadza też jakoś mechanizmy side effectów :) (W Angularze przy użyciu NgRx Store byłyby to effects)